video/out/opengl/video.c

```diff
@@ -3006,13 +3006,13 @@ void gl_video_render_frame(struct gl_video *p, struct vo_frame *frame,
     if (!has_frame || !mp_rect_equals(&p->dst_rect, &target_rc)) {
         struct m_color c = p->clear_color;
         float color[4] = {c.r / 255.0, c.g / 255.0, c.b / 255.0, c.a / 255.0};
-        p->ra->fns->clear(p->ra, target.tex, color, &target_rc);
+        p->ra->fns->clear(p->ra, target.tex, color);
```

Comment:
glClear without rendering would just be something else, either a separate call, or a new renderpass type.

I suggested this only because vulkan apparently has this builtin (?).